homeassistant/components/onkyo/manifest.json

```diff
@@ -2,6 +2,7 @@
   "domain": "onkyo",
   "name": "Onkyo",
   "documentation": "https://www.home-assistant.io/integrations/onkyo",
-  "requirements": ["onkyo-eiscp==1.2.7"],
-  "codeowners": []
+  "requirements": ["pyeiscp==0.0.3"],
+  "dependencies": [],
```

Comment:
```suggestion
```

Reply:
Can we leave this one out?
The minimal example in the [docs](https://developers.home-assistant.io/docs/creating_integration_manifest/) does include dependencies as an empty array.

Reply:
We generally remove empty ones.

Its safe to remove, especially since it was added in this PR.